cypress/integration/pages/articles/tests.js

```diff
@@ -94,14 +94,18 @@ export const testsThatFollowSmokeTestConfig = ({
     });
 
     describe(`Article Body`, () => {
-      it('should render a H1, which contains/displays a styled headline', () => {
+      it('should render a H1 which has attributes `id` and `tabindex` and displays a styled headline', () => {
         cy.request(`${config[service].pageTypes.articles.path}.json`).then(
           ({ body }) => {
             const headlineData = getBlockData('headline', body);
-            cy.get('h1').should(
-              'contain',
-              headlineData.model.blocks[0].model.blocks[0].model.text,
-            );
+            cy.get('h1')
```

Comment:
This test is great. It'd be good to remove the cypress tests checking h1 values from the `articles/test.js`, `liveRadio/tests.js` and `frontpage/test.js` and instead add a single in the [cypress/integration/pages/testsForAllPages.js](https://github.com/bbc/simorgh/blob/203bd72862237314a2aa005d7960941866f85f41/cypress/integration/pages/testsForAllPages.js)

This is since it'd ensure we're checking this attribute for all new pagetypes, e.g. the new `mediaAssetPage` as well. It'd make it easier to maintain too.

Reply:
Makes sense.